Driverless AI/machine-learning-experiment-scoring-and-analysis-financial-focus/machine-learning-experiment-scoring-and-analysis-financial-focus.md

```diff
@@ -1252,59 +982,56 @@ We are going to use this new experiment to run a new diagnostics test. You will
   2. Click on **Test Dataset** then select the **freddie_mac_500_test** dataset
   3. Initiate the diagnostics model by clicking on **Launch Diagnostics** 
```

Comment:
add : at the end of the sentence (every time an image follows from a sentence referring an image a : I added to notify the users of their connection)